onnx/defs/schema.h

```diff
@@ -506,7 +483,7 @@ class OpSchemaRegistry final {
       // Increase the highest version when you make BC-breaking changes to the
       // operator schema on specific domain. Update the lowest version when it's
       // determined to remove too old version history.
-      map_[ONNX_DOMAIN] = std::make_pair(1, 6);
+      map_[ONNX_DOMAIN] = std::make_pair(1, 7);
```

Comment:
Wait, as we discussed, we just keep using 6 right?

Reply:
technically, it's risky to reuse 6. as in the time in-between, there could be models using the other operators with consumed_input with version 6. :) I'm changing to use version 6 though.